app/src/main/res/layout/fragment_book_restaurant.xml

```diff
@@ -0,0 +1,154 @@
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/tv_book_restaurant_people_count" />
+
+        <ImageView
+            android:id="@+id/im_book_restaurant_people_count_add"
+            android:layout_width="wrap_content"
+
+
+            android:layout_height="wrap_content"
+            android:src="@drawable/ic_member_add"
+            app:layout_constraintBottom_toBottomOf="@+id/tv_book_restaurant_people_count"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/tv_book_restaurant_people_count"
+            app:layout_constraintTop_toTopOf="@+id/tv_book_restaurant_people_count" />
+
+        <TextView
+            android:id="@+id/tv_book_restaurant_people_count"
+            style="@style/TextAppearance.AppCompat.Large"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/spacing_extra_large"
+            android:text="1"
+            app:layout_constraintBottom_toTopOf="@+id/button_book_restaurant_continue"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/margin"
+            android:layout_width="0dp"
+            android:layout_height="60dp"
+            android:layout_marginRight="36dp"
+            app:layout_constraintTop_toBottomOf="@+id/tv_book_restaurant_restaurant_locality"
+            app:cardElevation="0dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            android:layout_marginTop="24dp"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toStartOf="@+id/tv_book_restaurant_restaurant_locality"
+            tools:layout_editor_absoluteY="115dp">
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+
+            <ImageView
+                android:id="@+id/iv_book_restaurant_calendar_icon"
+
+                android:layout_width="60dp"
+
+                android:layout_height="match_parent"
+                android:src="@drawable/ic_calendar_grey"/>
+
+
+                    <LinearLayout
+                        android:layout_toEndOf="@+id/iv_book_restaurant_calendar_icon"
+                        android:id="@+id/container_book_restaurant_date_picker"
+                        android:layout_marginStart="12dp"
+                        android:layout_width="match_parent"
+
+                        android:layout_height="match_parent"
+                        android:orientation="horizontal">
+
+
+                    </LinearLayout>
+            </RelativeLayout>
+        </androidx.cardview.widget.CardView>
+
+        <Button
+            android:id="@+id/button_book_restaurant_continue"
```

Comment:
Could reuse the button style we've been using. Check out https://github.com/checkin-team/Checkin-Android/blob/master/app/src/main/res/layout/activity_shop_join.xml